### PR TITLE
TD-26 add 3D Viewer

### PR DESCRIPTION
**Pull request by NadyaG**
* Added mentions of 3D Viewer to all sections.
* Made the query parameters list into a table, with +/- showing for which component each parameter is supported.
* Noted that the events for current model make no sense for 3D Viewer.
* Noted that 3D Viewer would work just as well on a desktop.
* DIDN'T remove the mobile/desktop structure of the guide - should I?

### CODE REVIEW

File: README.md
Comment (olgati) [inline README.md:26]: We can keep Virtual Fitting Room here. We use it as a generic name for the solution that can present either virtual tey-on or 3D Viewer

File: README.md
Comment (olgati) [inline README.md:47]: Here we need to change Virtual Fitting Room -> Virtual Try-On (see the comment above)

File: README.md
Comment (olgati) [inline README.md:49]: The same: Fitting Room -> Virtual Try-On

File: README.md
Comment (olgati) [inline README.md:51]: modelid is mandatory for a 3D Viewer. We need to add this notice

File: README.md
Comment (olgati) [inline README.md:71]: Again, Virtual Fitting Room -> Virtual Try-On

File: README.md
Comment (olgati) [inline README.md:105]: Can we add here sample images from the slide https://docs.google.com/presentation/d/1DMPs6F33LOCvE0twhrVQXIcGRTf_4LGKoJl0pvDjG8k/edit#slide=id.g1fcb4a2b870_0_25

**Review by olgati — CHANGES_REQUESTED** [review]

File: README.md
Comment (NadyaG) [reply README.md:51]: It says **required** in the first column

Comment (olgati) [reply README.md:51]: It is required only for a 3D Viewer setup. For VTO this parameter is optional.

File: README.md
Comment (NadyaG) [reply README.md]: Done, converting to a regular PR and waiting for the final review.

**Review by vpenyazkov — APPROVED** [review]

**Review by olgati — APPROVED** [review]

**Pull request merged**
